apps/src/code-studio/pd/application/teacher/TeacherApplication.jsx

```diff
@@ -88,6 +87,9 @@ const TeacherApplication = props => {
     if (nominated) {
       url += '?nominated=true';
     }
+    if (savedStatus === 'incomplete') {
+      url += '?returned=true';
+    }
```

Comment:
might want to upgrade this to use https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams or something similar or else you'd potentially end up with ?nominated=true?returned=true which isn't a valid url

Reply:
oh the query-string library already being used there will take care of it too

Reply:
Ahh good to know. What *should* the url look like if it's both nominated and savedStatus? I was thinking I don't want to change the nominated one if possible

Reply:
Or should this be different? Seems like the default is the `&`: `foo=bar&abc=xyz&abc=123` (https://nodejs.org/api/querystring.html#querystringparsestr-sep-eq-options)

Reply:
it'd get an `&` between each query param, so `?nominated=true&returned=true` or `?returned=true` or `?returned=true&nominated=true`

Reply:
Not sure who to contact about Google Analytics if we change it ...

Reply:
Okay, I think that's easy for GA people to change ... will update!

Reply:
shouldn't be anything that needs to change on the GA side! this is a standardized way of constructing urls

Reply:
Ah, cool beans. Just updated it! Might should add test coverage for this now ... let me know what you think, and I'll check in the morning.